docs/api/spec/pk-concepts.xml

```diff
@@ -695,6 +695,10 @@
             <entry><literal>desktop-kde</literal></entry>
             <entry>KDE</entry>
           </row>
+          <row>
+            <entry><literal>desktop-dde</literal></entry>
+            <entry>DDE</entry>
```

Comment:
```suggestion
            <entry>Deepin</entry>
```

Reply:
Why? DDE is the name that we are supposed to use. DDE is the desktop environment's name, *deepin* is the distribution and branding name.